Composer/packages/client/src/pages/setting/publisher/index.js

```diff
@@ -187,9 +189,11 @@ const CreatePublishTarget = props => {
 
   const submit = () => {
     try {
+      // validate that JSON parses correctly
       JSON.parse(config);
       return props.onSave(name, targetType, config);
     } catch (err) {
+      console.error(err);
       alert('Error parsing configuration');
```

Comment:
this alert isn't in the diff, however do we really plan to use the vanilla `alert` control?

Reply:
It's outdated publish page, I remove this file.

Reply:
outdated pages, I remove it